examples/tutorials/research/01_introduction_to_research.ipynb

```diff
@@ -0,0 +1,1012 @@
+{
```

Comment:
<p><code>train_root</code> looks a bit overcomplicated here.</p><p>Add <code>run_later</code> to <code>train_template</code> and then <code>train_pipeline = train_template &lt;&lt; mnist.train</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/337/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>